controllers/terminal/controllers/terminal_controller.go

```diff
@@ -444,6 +446,14 @@ func buildLabelsMap(terminal *terminalv1.Terminal) map[string]string {
 	return labelsMap
 }
 
+func getDomain() string {
+	domain := os.Getenv("DOMAIN")
```

Comment:
Seems we can store env variabe `DMOAIN` at init time and just read it var at every reconceil call to save some system calls.

Reply:
done